test/of.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var M = require('..');
```

Comment:
We could attach `Just` and `Nothing` to `Maybe`. The downside would be we'd then have both `S.Nothing` and `S.Maybe.Nothing` in Sanctuary. It may not be a good idea after all. Let's stick with `M` for now. :)